### PR TITLE
Gerry larios add votes 24

### PR DESCRIPTION
**Pull request by GerryLarios**
# PR Description

✨ This PR fixes the issue number: #24
 
✨ Here's the summary of my changes and my motivation behind them:

- Add vote model migration
- Add a relationship to the user and vote models
- Add a relationship to the definition and vote models
- Add vote controller with the actions new, create, edit, update, destroy
- Add resource routes
- Add votes to `db/seeds.rb`

✨ This PR changes or adds functionality which will require updating the documentation: YES/NO (leave what's relevant)
YES

# Self-check:

Please check all that apply:

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my code, refactored it to the best of my abilities, and deleted all unnecessary code
- [x] I have commented on my code, particularly in hard-to-understand areas; my comments are concise
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings and no new bugs

✨ ✨ ✨  Thank you for contributing to the Code Bending Dictionary ✨ ✨ ✨

### CODE REVIEW

**Comment by GerryLarios** [comment]
Hello @bigfishh and @HyeokJungKim , let me know if I need to change something.

File: app/controllers/votes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  def new
+    @vote = Vote.new
+  end 
+
+  def create
+    @vote = Vote.create(vote_params)
+  end
+
+  def edit;end
```
Comment (HyeokJungKim) [inline app/controllers/votes_controller.rb]: @GerryLarios 
Would it be alright to take this line out to be consistent with the show action?

Comment (GerryLarios) [reply app/controllers/votes_controller.rb]: Sure thing! let me remove it.

File: app/controllers/votes_controller.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  def update
+    @vote = Vote.create(vote_params)
```
Comment (HyeokJungKim) [inline app/controllers/votes_controller.rb]: @GerryLarios 
It looks like you're creating a Vote instance here, rather than updating it.

**Comment by GerryLarios** [comment]
@HyeokJungKim, the comments were addressed.

**Comment by GerryLarios** [comment]
@sylwiavargas  @HyeokJungKim @bigfishh Hello, this is a friendly reminder about the review status. Do you have feedback about this PR?

**Comment by sylwiavargas** [comment]
Hi @GerryLarios , I'm sorry you had to wait so long. We had an unforeseeable situation. Thank you for your contribution, I have just merged it and added the appropriate label!

**Comment by sylwiavargas** [comment]
one more thing, @GerryLarios  — would you like to be added to our contributors list in the READme?
If so:
1. Would you give me please your name or nickname you wish to be featured
2. We usually link the github pic — is that ok?

**Comment by GerryLarios** [comment]
@sylwiavargas thanks!
About the contributors' list, you can use my nickname `Gerry Larios` and it's ok if you use my GitHub pic.

**Pull request merged**